### PR TITLE
Bump to 3.8.0 RC3

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
RC3 of library will be released soon.

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: fastlane/metadata/android/en-US/changelogs/30080053.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+3.8.0 RC2 (August, 21, 2019)
+
+- FIDO U2F support on login
+- load only 60 images on photo view, then load more on demand
+- do not auto upload .thumbnail files
+- allow to send crash report via email
+- paste clipboard into Collabora
+- use Conscrypt to support TLS1.3
+- show sharees in list view
+- bugfixes found by RC1
```
Comment (AndyScherzinger) [inline fastlane/metadata/android/en-US/changelogs/30080053.txt]: RC1/RC2!

File: fastlane/metadata/android/en-US/changelogs/30080053.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+3.8.0 RC2 (August, 21, 2019)
```
Comment (AndyScherzinger) [inline fastlane/metadata/android/en-US/changelogs/30080053.txt]: RC3!

Comment (AndyScherzinger) [reply fastlane/metadata/android/en-US/changelogs/30080053.txt]: plus correct date!

File: fastlane/metadata/android/en-US/changelogs/30080053.txt (hunk 1 above)
Comment (tobiasKaminsky) [reply fastlane/metadata/android/en-US/changelogs/30080053.txt]: In RC3 I fixed only the activity thing, which I mentioned separate.

File: fastlane/metadata/android/en-US/changelogs/30080053.txt (hunk 2 above)
Comment (tobiasKaminsky) [reply fastlane/metadata/android/en-US/changelogs/30080053.txt]: Indeed :+1:

**Pull request merged**